tools/mrbayes/mrbayes.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0"?>
+<tool id="mrbayes" name="MrBayes" version="3.2.6">
+  <description>with options and commands</description>
+  <requirements>
+    <requirement version="3.2.1" type="binary">mb</requirement>
```

Comment:
Also not sure if this is currently required or not. The tool_dependencies.xml installs mb, but I'm not sure if this is properly checking for it or not